tests/integration/api_v3/http/blockchainAppsTokenMetadata.test.js

```diff
@@ -79,8 +80,8 @@ describe('Blockchain application tokens metadata API', () => {
 		expect(response.meta).toMap(metaSchema);
 	});
 
-	it('retrieves blockchain application off-chain metadata for tokens by global chainID', async () => {
-		const response = await api.get(`${endpoint}?chainID=00000001`);
+	it('retrieves blockchain application off-chain metadata for tokens chainID', async () => {
```

Comment:
Please apply the same changes requested above.

Reply:
Applied.